### PR TITLE
bug(refs DPLAN-1879): Add aria-live for accessibility

### PR DESCRIPTION
**Pull request by gruenbergerdemos**
### Ticket
DPLAN-1879

The aria-live attribute is used to indicate that an element will be updated, and the screen reader should announce the changes. The value "polite" means that the screen reader will wait until it has finished speaking before announcing updates.

### CODE REVIEW

**Review by salisdemos — APPROVED** [review]

**Comment by gruenbergerdemos** [comment]
Sonarcloud was not happy with the label outside of DpInput, so I added an option to hide the label in demosplan-ui.

File: client/js/components/procedure/publicindex/DpSearchProcedureMap.vue [hunk 1]
```diff
@@ -12,39 +12,39 @@
...
-        :class="prefixClass('c-proceduresearch__search-field')"
         ref="autocomplete"
         v-model="currentAutocompleteSearch"
+        :class="prefixClass('c-proceduresearch__search-field')"
+        data-cy="searchProcedureMapForm:procedureSearch"
+        label="value"
+        name="search"
+        :height="'34px'"
+        :options="autocompleteOptions"
+        :placeholder="Translator.trans('procedure.public.search.placeholder')"
```
Comment (salisdemos) [inline client/js/components/procedure/publicindex/DpSearchProcedureMap.vue:24]: alphabetic order please

File: client/js/components/procedure/publicindex/DpSearchProcedureMap.vue [hunk 2]
```diff
@@ -12,39 +12,39 @@
...
-        name="search"
         id="procedure_search"
-        :class="prefixClass('c-proceduresearch__search-field')"
         ref="autocomplete"
         v-model="currentAutocompleteSearch"
+        :class="prefixClass('c-proceduresearch__search-field')"
+        data-cy="searchProcedureMapForm:procedureSearch"
+        label="value"
+        name="search"
+        :height="'34px'"
```
Comment (salisdemos) [inline client/js/components/procedure/publicindex/DpSearchProcedureMap.vue:22]: You can simplify it like:
```suggestion
        height="34px"
```
can we somehow "unmagic" that Number?

Comment (gruenbergerdemos) [reply client/js/components/procedure/publicindex/DpSearchProcedureMap.vue]: > can we somehow "unmagic" that Number?

I had no good idea how. The height prop in DpAutocomplete is used for the `box-height` in `vue-omnibox`. 
Order done in [8c56701](https://github.com/demos-europe/demosplan-core/pull/3235/commits/8c56701cd9d111abc8ed4b25ef2292cff8b248ad)

**Review by salisdemos — APPROVED** [review]

**Pull request merged**